drones/image_processing/__init__.py

```diff
@@ -1 +1,2 @@
 import drones.image_processing.normalization as normalization
+from drones.image_processing.utils import calculate_focal, distance_to_camera
```

Comment:
I have no idea at the moment how to solve this problem with "No newline at end of file"